tck/jax-rs-signature-test/pom.xml

```diff
@@ -0,0 +1,92 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  Licensed to the Apache Software Foundation (ASF) under one or more
+  contributor license agreements.  See the NOTICE file distributed with
+  this work for additional information regarding copyright ownership.
+  The ASF licenses this file to You under the Apache License, Version 2.0
+  (the "License"); you may not use this file except in compliance with
+  the License.  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+  -->
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>org.apache.tomee</groupId>
+        <artifactId>tck</artifactId>
+        <version>10.0.0-SNAPSHOT</version>
+    </parent>
+
+    <artifactId>jax-rs-signature-test</artifactId>
+    <name>TomEE :: TCK :: JAX-RS Signature Test</name>
+
+    <properties>
+        <jaxrs.tck.version>3.1.3-1</jaxrs.tck.version>
+    </properties>
+
+    <repositories>
+        <!--TODO: We use the staging repository of jakarta as the tck dependencies are not yet pushed to central. Remove this asap.-->
+        <repository>
```

Comment:
Still needed or are they pushed to central now?

Reply:
No, it seems so be in central now: https://central.sonatype.com/artifact/jakarta.ws.rs/jakarta-restful-ws-tck

I removed the custom repositories.